### PR TITLE
chore: update nitro version to 2.3.3

### PR DESCRIPTION
**Pull request by TucksonDev**
Tested flow Orbit deployment UI + orbit-setup-script

### CODE REVIEW

**Review by Jason-W123 — APPROVED** [review]
Now nitro upgraded to v2.3.3-6a1c1a7

File: docker-compose.yaml [hunk 1]
```diff
@@ -86,15 +86,15 @@ services:
       service: proxy
 
   nitro:
-    image: offchainlabs/nitro-node:v2.3.0-3e14543
+    image: offchainlabs/nitro-node:v2.3.2-064fa11
```
Comment (Jason-W123) [inline docker-compose.yaml:89]:
```suggestion
    image: offchainlabs/nitro-node:v2.3.3-6a1c1a7
```

File: docker-compose.yaml [hunk 2]
```diff
@@ -86,15 +86,15 @@ services:
...
     ports:
       - "127.0.0.1:8449:8449"
     volumes:
       - "./config:/home/user/.arbitrum"
     command: --conf.file /home/user/.arbitrum/nodeConfig.json
 
   das-server:
-    image: offchainlabs/nitro-node:v2.3.0-3e14543
+    image: offchainlabs/nitro-node:v2.3.2-064fa11
```
Comment (Jason-W123) [inline docker-compose.yaml:97]:
```suggestion
    image: offchainlabs/nitro-node:v2.3.3-6a1c1a7
```

**Comment by TucksonDev** [comment]
Tested on 2.3.3 too

**Pull request merged**